docs/example-scenario/apps/internal-apim-ase-scenario.md

```diff
@@ -0,0 +1,193 @@
+   * Create a Cosmos DB service for web APIs hosted internal to VNET: noderestapidb
+   * Create DNS entries on the Private DNS zone created
+   * You could make use of Azure DevOps Pipelines to configure the agents on Virtual Machines to deploy the code for Web App on internal Network
+   * For testing the API App internally, create a test VM within the VNET subnet
+6. Creates API Management service: apim-internal
+7. Configures the service to connect to intern VNET on Subnet: apimsubnet. After the deployment is compelte, perform the below additional steps
+   * Configure custom domains for APIM Services using SSL Cert
+     * API portal (api.contoso.org)
+     * Dev Portal (portal.contoso.org)
+     * In the APIs section, configure the ASE Apps using ASE’s DNS name added Policy for HOST Header for the Webapp
+     * Use the above created test VM to test the API Management service internal on the Virtual Network
+> *Note: the testing the APIM APIs from Azure portal will still NOT work as we don’t have api.contoso.org not be able to publicly resolve*
+
+8. Configures Application Gateway (WAF V1) to access the APU service: apim-gateway on Port 80. Add SSL Certs to the App Gateway and corresponding Health probes and Http settings. Also configure the Rules and Listeners to use SSL Cert
+
+Once the above steps are successfully completed, Configure the DNS entries in Godaddy CNAME entries of api.contoso.org and portal.contoso.org with App Gateway’s public DNS name: ase-appgtwy.westus.cloudapp.azure.com and verify if you are able to reach the Dev Portal from Public and are able to test the APIM services APIs using Azure Portal
+
+*Note that it is not a good practice to use same URL for Internal and External Devs for the APIM services (currently in the above demo, both URLs are same). If we want to choose to have different URLs for internal and external devs, we could make use of App Gateway WAF v2 which supports http redirection and much more.*
+
+## Pricing
+
+API Management is offered in four tiers: developer, basic, standard, and premium. You can find detailed guidance on the difference in these tiers at the [Azure API Management pricing guidance here.][apim-pricing]
+
+Customers can scale API Management by adding and removing units. Each unit has capacity that depends on its tier.
+
+> [!NOTE]
+> The Developer tier can be used for evaluation of the API Management features. The Developer tier should not be used for production.
+
+To view projected costs and customize to your deployment needs, you can modify the number of scale units and App Service instances in the [Azure Pricing Calculator][pricing-calculator].
+
+Similarly, the [App Service Environments pricing guidance is provided here][ase-pricing]
+
+Application Gateway pricing can be be [configured here][appgtwy-pricing] depending upon the required tier and resources 
+
+## Related resources
+
+Check out the related scenario on [Migrating legacy web APIs to API Management][related-scenario]
```

Comment:
What other information would this provide?

Reply:
It is just another scenario which involves migrating legacy web APIs to API Management